### PR TITLE
Fix/45/error state not updates

### PR DESCRIPTION
**Pull request by svettwer**

### CODE REVIEW

File: packages/sakuli-forwarder-prometheus/src/gauge.utils.spec.ts [hunk 1]
```diff
@@ -27,246 +33,339 @@ describe("gauge utils", () => {
...
+
+            //GIVEN
+            const caseContextMock = mockPartial<TestCaseContext>({
+                id: "Case Context Mock",
+                kind: "case"
+            });
+            const stepContextMock = mockPartial<TestStepContext>({
+                id: "Step Context Mock",
+                kind: "step",
+                error: new Error()
```
Comment (cliffle) [inline packages/sakuli-forwarder-prometheus/src/gauge.utils.spec.ts]: This kinda confuses me. The state of the step is not an error but it contains an error?
For the register case without error state (line 179), you didn't add an error to the caseContextMock.  :confused:

File: packages/sakuli-forwarder-prometheus/src/gauge.utils.spec.ts [hunk 2]
```diff
@@ -27,246 +33,339 @@ describe("gauge utils", () => {
...
+        it("should register action error gauge without error state", () =>{
+            //GIVEN
+            const stepContextMock = mockPartial<TestStepContext>({
+                id: "Step_Context_Mock",
+                kind: "step"
+            });
+            const actionContextMock = mockPartial<TestActionContext>({
+                id: "Action Context Mock",
+                kind: "action",
+                error: new Error()
```
Comment (cliffle) [inline packages/sakuli-forwarder-prometheus/src/gauge.utils.spec.ts]: Same here

File: packages/sakuli-forwarder-prometheus/src/gauge.utils.spec.ts (hunk 1 above)
Comment (svettwer) [reply packages/sakuli-forwarder-prometheus/src/gauge.utils.spec.ts]: Good finding! That is just a leftover from an experiment during development. :smile:

File: packages/sakuli-forwarder-prometheus/src/gauge.utils.spec.ts (hunk 2 above)
Comment (svettwer) [reply packages/sakuli-forwarder-prometheus/src/gauge.utils.spec.ts]: Same here

**Comment by svettwer** [comment]
Sorry for the delay... Fixing this property parsing test setup was a little tricky. :sweat_smile:

**Review by cliffle — APPROVED** [review]
:+1:

**Pull request merged**